src/fixed-footer-layout.tsx

```diff
@@ -114,26 +117,20 @@ const FixedFooterLayout = ({
     ]);
 
     /**
-     * Notes about the background:
-     *
-     * - If a gradient is used as background color, the end color of the gradient must start at the
-     *   top of the fixed footer. That means that the gradient height is "viewable area" minus "footer height".
-     *
-     * - The content could be scrollable and the gradient must scroll with it, so an additional background is
-     *   needed to fill the gap between the gradient and the fixed footer. A fullscreen fixed div does this job.
-     *
-     * - When there is not enough vertical space, instead of a fixed footer, the footer is placed at the
-     *   bottom of the content. In this case, the background size is the same as the content (height: 100%).
-     *
-     * - A feedback screen could contain a navigation bar. The topDistance is needed to calculate the gradient height.
-     *
-     * - Using a Portal because the background must be rendered outside the content container
+     * Diagram of the layout:
+     * https://excalidraw.com/#json=No0s6LB7QO735nv-wGIEP,9OOuqiaFInbtr1YjMm4g4Q
      */
     const renderBackground = () => {
         return (
             <Portal>
-                <div className={styles.fixedBackgroundLayer} style={{background: footerBgColor}} />
                 <div
+                    className={styles.fixedBackgroundLayer}
+                    style={{
+                        background: `linear-gradient(180deg, ${topColor} 0%,${topColor} 50%, ${footerBgColor} 50.1%, ${footerBgColor} 100%)`,
```

Comment:
This should be equivalent:
```suggestion
                        background: `linear-gradient(180deg, ${topColor} 50%, ${footerBgColor} 50% 100%)`,
```

Reply:
TIL